### PR TITLE
Add documentation of solar thermal collectors facades

### PR DESCRIPTION
**Pull request by MaGering**
This PR adds a description of facades to the documentation of solar thermal collector.

https://oemof-thermal.readthedocs.io/en/features-docs-solar-thermal-collector-facade/solar_thermal_collector.html

The structure is based on PR https://github.com/oemof/oemof-thermal/pull/98

### CODE REVIEW

File: docs/solar_thermal_collector.rst [hunk 1]
```diff
@@ -87,40 +84,110 @@ These arguments are used in the formulas of the function:
...
-Please see the API for all parameters, which have to be provided. The needed dataframe
-must hold columns for a date, the ambient temperature and the irradiance. Some of the
-parameters which have to be provided for the precalculation define the column names
-of the dataframe.
+Usage
+_____
+
+It is possible to use the precalculation function as stand-alone function to calculate the collector values
+:math:`\dot Q_{coll}`, :math:`\eta_C` and :math:`E_{coll}`. Or it is possible
+to use the ParabolicTroughCollector facade to model a collector with further
```
Comment (MaGering) [inline docs/solar_thermal_collector.rst]:
```suggestion
to use the SolarThermalCollector facade to model a collector with further
```

File: docs/solar_thermal_collector.rst [hunk 2]
```diff
@@ -87,40 +84,110 @@ These arguments are used in the formulas of the function:
...
+losses (e.g. in pipes or pumps) and the electrical consumption of pipes within a single step.
+Please note: As the unit of the input irradiance is given as power per area,
+the outputs :math:`\dot Q_{coll}` and :math:`E_{coll}` are given in the same
+unit. If these values are used in an oemof source, the unit of the nominal
+value must be an area too.
+
+
+Solar thermal collector precalculations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Please see the :ref:`api of the solar_thermal_collector module <api_label>` for all parameters which have to be provided, also the ones that are not part of the described formulas above. The data for the irradiance and the ambient temperature must have the same time index. Be aware of the correct time index regarding the time zone, as the utilized pvlib need the correct time stamp corresponding to the location.
```
Comment (MaGering) [inline docs/solar_thermal_collector.rst]:
```suggestion
Please see the :ref:`api of the solar_thermal_collector module <api_label>` for all parameters which have to be provided, also the ones that are not part of the described formulas above. The data for the irradiance and the ambient temperature must have the same time index. Be aware of the correct time index regarding the time zone, as the utilized pvlib needs the correct time stamp corresponding to the location.
```

File: docs/solar_thermal_collector.rst [hunk 3]
```diff
@@ -87,40 +84,110 @@ These arguments are used in the formulas of the function:
...
+    	irradiance_diffuse=input_data['diffuse_horizontal_W_m2'],
+    	temp_amb_col=input_data['temp_amb'],
+    )
+
+It calculates the heat of the collector in the same
+way as the calculations do. Additionally, it integrates the calculated heat as an input
+into a component, uses an electrical input for pumps and gives a heat output,
+which is reduced by the defined additional losses. See flat_plate_collector_example_facade.py for an application example. It models the same
+system as the flat_plate_collector_example.py, but uses the SolarThermalCollector facade
+instead of separate source and transformer.
```
Comment (MaGering) [inline docs/solar_thermal_collector.rst]: This paragraph is a duplicate. Same information are in line 146 - 156.

Comment (c-moeller) [reply docs/solar_thermal_collector.rst]: I deleted it.

**Pull request merged**